Patches/Medical System Expansion/HediffDefs/Hediffs_AddedParts/Patch_Hediffs_AddedParts_VanillaArchotech.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+			<!-- ==== Checking for the mod ==== -->
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>Medical System Expansion</modName>
+			</li>
+
+			<!-- ==== Vanilla Archotech Prosthesis  ==== -->
+			<!-- ArchotechEye -->
+			<li Class="PatchOperationTest">
+				<xpath>Defs/HediffDef[defName="ArchotechEye"]/stages</xpath>
+				<success>Invert</success>
+			</li>
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/HediffDef[defName="ArchotechEye"]</xpath>
+				<value>
+					<stages>
+						<li>
+							<minSeverity>0.000001</minSeverity>
+							<statOffsets>
+								<ArmorRating_Sharp>0.026</ArmorRating_Sharp>
+								<ArmorRating_Blunt>0.048</ArmorRating_Blunt>
+								<ArmorRating_Heat>0.004</ArmorRating_Heat>
+							</statOffsets>
+						</li>
+					</stages>
+				</value>
+			</li>
+			<!-- ArchotechArm -->
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/HediffDef[defName="ArchotechArm"]</xpath>
+				<value>
+					<stages>
+						<li>
+							<minSeverity>0.000001</minSeverity>
+							<statOffsets>
+								<ArmorRating_Sharp>0.786</ArmorRating_Sharp>
+								<ArmorRating_Blunt>1.444</ArmorRating_Blunt>
+								<ArmorRating_Heat>0.105</ArmorRating_Heat>
+							</statOffsets>
+						</li>
+					</stages>
+				</value>
+			</li>
+			<!-- ArchotechLeg -->
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/HediffDef[defName="ArchotechLeg"]</xpath>
+				<value>
+					<stages>
+						<li>
+							<minSeverity>0.000001</minSeverity>
```

Comment:
What's the rationale for these minSeverity? Any hediff stage without this field is automatically the default stage, which seems to be what you want.

Reply:
Ah I get it, it's needed to play nice with MSE. Fair.

Reply:
yeah, if i didn't have <minSeverity />, the game would complain about stages not being in the order of minimal severity. which is dumb, because that's the only stage for that hediff.